rpcs3/Emu/VFS.cpp

```diff
@@ -179,6 +184,11 @@ bool vfs::unmount(std::string_view vpath)
 		}
 	};
 	unmount_children(table.root, 0);
+	const auto mp = lv2_fs_object::get_mp(vpath);
+	if (mp == &g_mp_sys_dev_usb)
+		mp->is_mounted &= ~(1U << vpath.back() - '0');
+	else
+		mp->is_mounted = false;
```

Comment:
this is not a boolean

Reply:
I use it to store the mount status of 8 dev_usbXXX devices, for the other devices, it is used as a boolean.